packages/build-info/package.json

```diff
@@ -52,7 +52,7 @@
     "minimatch": "^9.0.0",
     "read-pkg": "^9.0.0",
     "semver": "^7.3.8",
-    "yaml": "^2.1.3",
+    "yaml": "^2.8.0",
```

Comment:
Unrelated change, just matching the version to what's in the package-lock already.
See https://github.com/netlify/build/pull/6332